### DISCUSSION TITLE
About the easter egg and arachnophobia

### DISCUSSION THREAD

**Discussion by enzocano** [ParabolInc/parabol#4875] [Feedback]
Hi folks!
The recently introduced easter egg (https://github.com/ParabolInc/parabol/pull/4800) should be totally optional or you should provide some mechanism to deactivate it since it can make someone with arachnophobia uncomfortable. Sadly, this is not a hypothetical example, I just had a meeting and one person was freaked out by this.

**Comment by tianrunhe** [comment]
Hi Enzo, I'm sorry our Easter egg freaks people out! I just did an emergency deployment to disable it (you can see the change log [here](https://github.com/ParabolInc/parabol/blob/master/CHANGELOG.md#changed)).

Thanks for your feedback and once again, our sincere apology from Parabol team!

**Comment by jordanh** [comment]
Resolved last year, about a week after April Fool's day. Sorry about the scare!
